core/templates/dev/head/domain/question/QuestionObjectFactory.js

```diff
@@ -99,7 +99,12 @@ oppia.factory('QuestionObjectFactory', [
         pendingMisconceptionNamesToTag.forEach(function(misconceptionName) {
           returnString = returnString + ' ' + misconceptionName + ',';
         });
-        return returnString.slice(0, -1);
+        returnString = returnString.slice(0, -1);
+        returnString = returnString + '. \nClick on (or create) an answer ' +
```

Comment:
Just to check, do you need the newline char / does it show up properly?

Reply:
No, it doesn't show up, removed it.